### ISSUE TITLE
composer install cmb2/cmb2 work wrong

### ISSUE THREAD

**Issue by cehojac** [CMB2/CMB2#1269]
## Expected Behavior:

When execute composer for install cmb2/cmb2 create a wp-content/plugins/cm2 folder in the main path and not in vendor/cmb2
## Actual Behavior:

Before
/
|-vendor (empty)

composer file:
```json
 "require": {
        "cmb2/cmb2": "dev-master",
        "composer/installers": "v1.0.12"
    },
"extra": {
        "installer-paths": {
        "vendor/{$name}/": ["cmb/cmb2"] 
        }
    }
```
Actual Result
/
|-vendor
|--composer
|-wp-content 
|--plugins
|---cmb2 (the cmb2 content)

## Possible Solution

When execute composer store te cmb2 in vendor folder

/
|-vendor
|--composer
|--cmb2 (the cmb2 content)

**Comment by salcode** [comment]
@cehojac 

I had this same problem. My issue came up when I also required the `composer/installers` package (I'm guessing you're requiring this package too).

The solution was to add 

```
    "extra": {
        "installer-paths": {
            "vendor/cmb2/cmb2": ["cmb2/cmb2"]
        }
    }
```

to my `composer.json`.

I have more information about how I figured this out in my blog post, [CMB2 as a Dependency with Composer](https://salferrarello.com/cmb2-composer-dependency/#composer-extra).
